broker/src/tunneldigger_broker/tunnel.py

```diff
@@ -243,7 +243,7 @@ def close(self, reason=protocol.ERROR_REASON_UNDEFINED):
         :param reason: Reason code for the tunnel being closed
         """
 
-        logger.info("Closing tunnel %d (%s) after %d seconds", self.tunnel_id, self.uuid, time.time() - self.created_time)
+        logger.info("Closing tunnel {} ({}) after {} seconds (reason=0x{:x})".format(self.tunnel_id, self.uuid, int(time.time() - self.created_time), reason))
```

Comment:
Nit: Don't pre-format in logging. :-)

Reply:
I know you prefer the `%` style but hey formatting is so much easier with the modern syntax...

Reply:
I know you prefer the % style but hex formatting is so much easier with the modern syntax...

Reply:
True. :-)

You could also do:

```
logger.info("Closing tunnel %d (%s) after %d seconds (reason=%s)", self.tunnel_id, self.uuid, time.time() - self.created_time, "0x{:x}".format(reason))
```